modules/flash-script.nix

```diff
@@ -47,7 +47,29 @@ in
 
           edk2NvidiaPatches = mkOption {
             type = types.listOf types.path;
-            default = [];
+            default = [ ];
+          };
+
+          secureBoot = {
+            enrollDefaultKeys = lib.mkEnableOption "default UEFI keys";
+            defaultPkEslFile = mkOption {
+              type = lib.types.path;
+              description = lib.mdDoc ''
+                The path to the UEFI PK public key in ESL format.
```

Comment:
I think to be more accurate this should just say "The path to the UEFI PK EFI Signature List (ESL)"